lib/ExpensiMark.js

```diff
@@ -282,8 +282,8 @@ export default class ExpensiMark {
             },
             {
                 name: 'blockquoteopen',
-                regex: /<blockquote>/gi,
-                replacement: ' ',
+                regex: /([^\s-])(<blockquote>)/gi,
```

Comment:
why is the change needed, and what does it do?

Reply:
That's rules are used to solve this https://github.com/Expensify/App/issues/8134.

Basically replacing the line breaks before the blockquote with a space. Since we get rid of the `<br />` we need to add a new rule to add a space before blockquote (if there's a string before `<blockquote>`).